From 7e17d3243a92a1b281a904d2623a6fc956186cd3 Mon Sep 17 00:00:00 2001
From: Steffen Trumtrar <s.trumtrar@pengutronix.de>
Date: Mon, 27 May 2013 12:33:05 +0000
Subject: [PATCH 0772/1115] video: display_timing: make parameter const

As the device_node pointer is not changed in of_get_display_timing and
parse_timing_property it can be a const pointer.

Signed-off-by: Steffen Trumtrar <s.trumtrar@pengutronix.de>
Signed-off-by: Dave Airlie <airlied@redhat.com>
(cherry picked from commit f583662347c62d80ac916658ae491274621c88ca)

Conflicts:
	drivers/video/of_display_timing.c
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/video/of_display_timing.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/video/of_display_timing.c b/drivers/video/of_display_timing.c
index 9c0f17b..3fc6ff0 100644
--- a/drivers/video/of_display_timing.c
+++ b/drivers/video/of_display_timing.c
@@ -23,7 +23,7 @@
  * Every display_timing can be specified with either just the typical value or
  * a range consisting of min/typ/max. This function helps handling this
  **/
-static int parse_timing_property(struct device_node *np, const char *name,
+static int parse_timing_property(const struct device_node *np, const char *name,
 			  struct timing_entry *result)
 {
 	struct property *prop;
-- 
1.7.5.4

